ct/bambuddy.sh

```
@@ -0,0 +1,78 @@
+    systemctl stop bambuddy
+    msg_ok "Stopped Service"
+
+    msg_info "Backing up Configuration and Data"
+    cp /opt/bambuddy/.env /opt/bambuddy.env.bak
+    cp -r /opt/bambuddy/data /opt/bambuddy_data_bak
+    msg_ok "Backed up Configuration and Data"
+
+    CLEAN_INSTALL=1 fetch_and_deploy_gh_release "bambuddy" "maziggy/bambuddy" "tarball" "latest" "/opt/bambuddy"
+
+    msg_info "Updating Python Dependencies"
+    cd /opt/bambuddy
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)